src/components/navigation/header/config/iconItems.ts

```diff
@@ -27,4 +30,15 @@ export const iconItems = ({
           }),
       }
     : null,
+  favorites: isLoggedIn
```

Comment:
Not 100% sure about this but I believe you can solve this by returning `visibilitySettings: { userType: {...}}` from the link config instead. I am not sure it's needed to pass through isLoggedIn